src/Planners/include/BipedalLocomotion/Planners/SO3Planner.h

```diff
@@ -0,0 +1,159 @@
+/**
+ * @file SO3Planner.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_PLANNERS_SO3_PLANNER_H
+#define BIPEDAL_LOCOMOTION_PLANNERS_SO3_PLANNER_H
+
+#include <manif/SO3.h>
+
+#include <BipedalLocomotion/System/Advanceable.h>
+
+namespace BipedalLocomotion
+{
+namespace Planners
+{
+
+/**
+ * Trivialization used in the SO3Planner
+ */
+enum class LieGroupTrivialization
+{
+    Left,
+    Right
+};
```

Comment:
Just one small theoretical nudge, the trivialization usually applies to the tangent space, so I wouldn't call it `LieGroupTrivialization`.  Instead maybe 'LieGroupVelocityTrivialization' or `GroupVelocityTrivialization` or `VelocityTrivialization` might be possible suggestions. 

cc @traversaro  please correct me if I am wrong.

Reply:
Indeed trivialization is more related to the element of the tangent space of the lie group, but I don't think that even `LieGroupTrivialization` is confusing, it is basically a sort of [metonymy](https://en.wikipedia.org/wiki/Metonymy). Personally all the proposed option seems ok for me.